src/object/srv_obj.c

```diff
@@ -943,126 +1022,43 @@ ds_obj_rw_handler(crt_rpc_t *rpc)
 		 *	modification or not, so still need to dispatch
 		 *	the RPC to other replicas.
 		 */
```

Comment:
the above comment for "non-replicated object" is inconsistent with the new code after your change. since L979 the non-replicated update is already handled there.
But I am not sure do we really need resend check for non-replicated obj update?

Reply:
ah, right, we are still discussing which way for resend. For now, I would not touch these comment. Once we come out a conclusion, I will add them back.